app/lib/actions/getPharmacy.js

```diff
@@ -0,0 +1,18 @@
+const apiRequest = require('../etl-toolkit/apiRequest');
+const fixPhoneNumbers = require('../fixPhoneNumbers');
+const config = require('../config');
+
+function toObject(pharmaString, odsCode) {
+  const pharmacy = JSON.parse(pharmaString);
+  // eslint-disable-next-line no-underscore-dangle
+  pharmacy._id = odsCode;
```

Comment:
I've thought some more about this and think it's appropriate to allow the ID fiel to be set on the etlStore. Setting the '_id' field from another key can be provided within the mongodb-updater.